src/routes/index.svelte

```diff
@@ -50,28 +49,24 @@
     <Flow />
     <div class="md:w-1/2 2xl:w-1/3">
       <h5 class="">THE PROBLEM</h5>
-      <h2 class="text-3xl font-semibold md:text-5xl md:mt-16">No need to re-invent the wheel</h2>
-      <p class="mt-4 md:mt-16">
-        With hyper, you can separate your core features from the necessary implementation details of high scale solutions. hyper 
-        provides a service boundary so your business logic stays in the application layer while you leverage the robust core services
-        hyper provides.
-      </p>
+      <h2 class="text-3xl font-semibold md:text-5xl md:mt-16">Technical Debt</h2>
+      <p class="mt-4 md:mt-16">What do developers hate the most about software development? Many will say technical debt, having to work
+      through so much noise to implement a simple change request.</p>
     </div>
   </section>
   <img class="md:hidden" src="/flow1.svg" />
   <section class="flex w-full pl-4 md:pt-44 md:pl-24 md:pr-24 md:mb-96">
     <div class="md:w-1/2 2xl:w-1/3">
       <h5>OUR SOLUTION</h5>
       <h2 class="text-3xl font-semibold md:text-5xl md:mt-8">
-      Use a cloud-native service framework.
+      hyper a service framework 
       </h2>
       <p class="mt-12">
-      hyper is a cloud-native solution that can run on any cloud and its ports and adapter architecture lets you use any
-      backend service you prefer. You can get started using the hyper cloud service in less that a minute. 
+      hyper is a cloud-native service solution that gives you clean and easy wat to use APIs to create scalable performant services. With hyper you can get started building applications that scale quickly and keep <b>techical debt</b> low.  
```

Comment:
hyper is a cloud-native service solution delivering a straightforward way to leverage APIs to create scalable performant services. Create features quickly into apps that scale, all while keeping technical debt at bay.